README.md

```diff
@@ -69,7 +69,20 @@ frisby.create(...)
   .toss()
 ```
 
-When using `intercept()` or `networkOff()`, the plugin restores network access
+Mock one request only if a condition evaluates to `true`.
+
+```js
+const someCondition = true
+frisby.create(...)
+  .get(...)
+  .conditionalIntercept(someCondition, nock => nock('http://example.com')
```

Comment:
What do you think about calling this `interceptIf()`? Seems elegant. Though I don't feel strongly about it.